include/controller/c/webots/robot_window_utils/string_utils.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 1996-2021 Cyberbotics Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Description:  Defines generic string functions to parse the robot window messages.
+ */
+
+#ifndef STRING_UTILS_H
+#define STRING_UTILS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+char *string_utils_strsep(char **stringp, const char *delim);
+char *string_utils_replace(char *orig, char *rep, char *with);
```

Comment:
Those are generic string manipulation functions, not specific to the robot window. So, I would rename them with a `wbu_` prefix and would put them into the `utils` folder under the `string.h` filename.

Reply:
I added the `wbu_` prefix but I would not move them in the `utils` folder given that they should be distributed as a separate libraries and not included in the controller library.